llvm/lib/Transforms/IPO/FunctionSpecialization.cpp

```diff
@@ -924,16 +932,14 @@ bool FunctionSpecializer::findSpecializations(Function *F, unsigned FuncSize,
       }
       CodeSize += Visitor.getCodeSizeSavingsFromPendingPHIs();
 
+      unsigned CodeSizeSavings = getCostValue(CodeSize);
+      unsigned CodeSizeCost = FuncSize - CodeSizeSavings;
+
```

Comment:
I think we can sink these two back in the lambda where the TODO note was. I would rename CodeSizeCost to SpecSize.

Reply:
Nevermind, I was wrong since it's being used outside of the lambda.